### PR TITLE
[TEVA-2542] add variants for round 3 of cookie banner ab test

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-2542

mili/steven will be signing off using review env

variant urls:

* https://teaching-vacancies-review-pr-3449.london.cloudapps.digital/?ab_test_override[2021_05_cookie_consent_test]=bottom_sticky_default
* https://teaching-vacancies-review-pr-3449.london.cloudapps.digital/?ab_test_override[2021_05_cookie_consent_test]=bottom_sticky_gds
* https://teaching-vacancies-review-pr-3449.london.cloudapps.digital/?ab_test_override[2021_05_cookie_consent_test]=top_sticky_default
* https://teaching-vacancies-review-pr-3449.london.cloudapps.digital/?ab_test_override[2021_05_cookie_consent_test]=top_sticky_gds
* https://teaching-vacancies-review-pr-3449.london.cloudapps.digital/?ab_test_override[2021_05_cookie_consent_test]=top_static_default
* https://teaching-vacancies-review-pr-3449.london.cloudapps.digital/?ab_test_override[2021_05_cookie_consent_test]=top_static_gds

### CODE REVIEW

**Review by stevenleggdfe — COMMENTED** [review]
Looks good to me! Thanks Alex.

File: app/views/layouts/application.html.slim [hunk 1]
```diff
@@ -21,14 +21,12 @@ html.govuk-template.app-html-class lang="en" class=("modal" if current_variant?(
...
   body class=body_class data-vacancy_state=(@vacancy.present? ? @vacancy.state : "create")
-    - if current_variant?(:"2021_04_cookie_consent_test", :modal) && !cookies_preference_set?
-      .modal-container
     = render partial: "shared/google_tag_manager_body" if cookies["consented-to-cookies"] == "yes"
     = javascript_tag(nonce: true) do
       | document.body.className = ((document.body.className) ? document.body.className + " js-enabled" : "js-enabled");
     = render "shared/skip_links"
-    - unless cookies_preference_set? || %i[top none].include?(ab_variant_for(:"2021_04_cookie_consent_test"))
-      = render(AbTest::CookiesBannerComponent.new(create_path: create_cookies_preferences_path(cookies_consent: "yes", **utm_parameters), reject_path: create_cookies_preferences_path(cookies_consent: "no", **utm_parameters), preferences_path: cookies_preferences_path(**utm_parameters)).with_variant(ab_variant_for(:"2021_04_cookie_consent_test")))
+    - unless cookies_preference_set? || %i[bottom_sticky_default bottom_sticky_gds].include?(ab_variant_for(:"2021_05_cookie_consent_test"))
```
Comment (cpjmcquillan) [inline app/views/layouts/application.html.slim:28]: Sometimes I find this way round is easier to read in certain contexts (although I can never decide a clear cut, nicely defined rule for when to use in/include)
```ruby
ab_variant_for(:"2021_05_cookie_consent_test").in?(%i[bottom_sticky_default bottom_sticky_gds])
```

File: config/ab_tests.yml [hunk 2]
```diff
@@ -28,21 +28,24 @@ shared:
...
-    modal: 1
-    gds: 1
+  2021_05_cookie_consent_test:
+    bottom_sticky_default: 1
+    bottom_sticky_gds: 1
+    top_static_default: 1
+    top_static_gds: 1
+    top_sticky_default: 1
+    top_sticky_gds: 1
+  2021_04_application_preview_test:
```
Comment (cpjmcquillan) [inline config/ab_tests.yml:38]: Rebase rebase!

**Review by cpjmcquillan — APPROVED** [review]
You need to rebase on master, but otherwise lgtm

**Pull request merged**